src/checkout/views/Payment/Gateways/Stripe/Stripe.tsx

```diff
@@ -1,16 +1,35 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 import { Elements, StripeProvider } from "react-stripe-elements";
 
 import { ProviderProps } from "../../View";
 import CardForm from "./CardForm";
 
 export const Stripe = (props: ProviderProps) => {
+  const [stripe, setStripe] = useState(null);
+
   const apiKey = props.paymentGatewayConfig.find(
     ({ field }) => field === "api_key"
   ).value;
+
+  useEffect(() => {
+    if (!(window as Window).Stripe) {
+      const stripeJs = document.createElement("script");
+
+      stripeJs.src = "https://js.stripe.com/v3/";
```

Comment:
I think we should move the link somewhere up in hierarchy(for ex to src/core/config or sth similar